src/wasm/body.rs

```diff
@@ -13,6 +13,7 @@ use wasm_bindgen::JsValue;
 /// passing many things (like a string or vector of bytes).
 ///
 /// [builder]: ./struct.RequestBuilder.html#method.body
+#[derive(Clone)]
```

Comment:
We don't want to do this one, at least, since this makes a public API promise that the type will always implement `Clone`.